packages/select/src/Select/Select.tsx

```diff
@@ -0,0 +1,121 @@
+import { Combobox } from "@telegraph/combobox";
+import { TgphComponentProps } from "@telegraph/helpers";
+import React from "react";
+
+type Option = TgphComponentProps<typeof Combobox.Option>;
+
+type SingleValue = string;
+type MultipleValue = Array<string>;
+
+type RootProps = Omit<
+  TgphComponentProps<typeof Combobox.Root>,
+  "value" | "onValueChange" | "layout"
+> & {
+  size?: TgphComponentProps<typeof Combobox.Trigger>["size"];
+  triggerProps?: TgphComponentProps<typeof Combobox.Trigger>;
+  contentProps?: TgphComponentProps<typeof Combobox.Content>;
+  optionsProps?: TgphComponentProps<typeof Combobox.Options>;
+  multiple?: boolean;
+} & (
+    | {
+        multiple?: true;
+        value?: MultipleValue;
+        onValueChange?: (value: MultipleValue) => void;
+        layout?: TgphComponentProps<typeof Combobox.Root>["layout"];
+      }
+    | {
+        multiple?: false | undefined;
```

Comment:
I think the `| undefined` should be unnecessary since this is already an optional property: `multiple?`.

Reply:
Ah good catch, didn't delete this part when debugging.